etc/profile-a-l/imv.profile

```diff
@@ -0,0 +1,40 @@
+# Persistent local customizations
+include imv.local
+# Persistent global definitions
+include globals.local
+
+include allow-bin-sh.inc
+
+blacklist /usr/libexec
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-write-mnt.inc
+
+include whitelist-runuser-common.inc
```

Comment:
Can we use wvc, wrc, wusc?

Reply:
I thought I wanted to view images in var and usr.

Reply:
usr/share makes sense. var, IDK if it contains any images except flatpak icons, run can be added imho.